zcncore/transaction.go

```diff
@@ -1096,6 +1096,44 @@ func GetChainStats(ctx context.Context) (b *block.ChainStats, err error) {
 	return
 }
 
+func GetFeeStats(ctx context.Context) (b *block.FeeStats, err error) {
+
+	var numMiners = 4
+
+	if numMiners > len(_config.chain.Miners) {
+		numMiners = len(_config.chain.Miners)
+	}
+
+	var result = make(chan *util.GetResponse, numMiners)
+
+	queryFromMinersContext(ctx, numMiners, GET_FEE_STATS, result)
```

Comment:
please extend this func to work with miners and sharders
https://github.com/0chain/gosdk/blob/staging/zcncore/transaction_query.go#L316

Reply:
You want to add `miners` to `TransactionQuery` struct and have similar function for miners ?

Reply:
we can pass func here
```go
host, err := tq.getRandomMiner(ctx)
```
https://github.com/0chain/gosdk/blob/b4f7de4f984909c9e4a0af40379ff365bb2bde3f/zcncore/transaction_query.go#L328

Reply:
Whats the endpoint to check miners health ?

Reply:
@dabasov I have extended the ForAny  to work for both miners and sharders , do u want the GetFeeStats  to use this or are we good with current implementation of it .